components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -32,6 +33,7 @@
 
 
     <string name="error_malformedURI_title">The address isn’t valid</string>
+    <!-- LOCALIZATION NOTE (error_malformedURI_message) This string contains markup. The URL should not be localized. -->
```

Comment:
Just `<!-- This string contains markup. The URL should not be localized. -->`

Let's drop the gecko-isms about `LOCALIZATION NOTE`.